src/util/nativeStorageHelper.ts

```diff
@@ -1,52 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
 /* eslint-disable fp/no-class */
 import AsyncStorage from '@react-native-async-storage/async-storage'
 
-/**
- *
- */
-function handleError(func: string, param?: string): Promise<string> {
-  let message
-  if (!param) {
-    message = func
-  } else {
-    message = `${func}() requires at least ${param} as its first parameter.`
-  }
-  console.warn(message) // eslint-disable-line no-console
-  return Promise.reject(message)
-}
 
 /**
- *
+ * Class SyncStorage - used to made async store sync an
+ * prevent changing to async call the whole app.
  */
 class SyncStorage {
   data: Map<string, string> = new Map()
 
   loading = true
 
   async init(): Promise<void> {
-    await AsyncStorage.getAllKeys().then((keys: string[]) =>
-      AsyncStorage.multiGet(keys).then((data: any): any[] => {
-        data.forEach(this.saveItem.bind(this))
-
-        return [...this.data]
-      }),
-    )
+    const keys = await AsyncStorage.getAllKeys();
+    const data = await AsyncStorage.multiGet(keys) as string[][];
+    data.forEach(this.saveItem.bind(this));
 
+    // eslint-disable-next-line no-unused-expressions
+    [...this.data];
```

Comment:
Sorry, I'm still not understanding this line. Could you explain?

Reply:
I just removed this line, it wasn't actually doing anything 😅.

Reply:
Phew... my Javascript understanding was about to break 😂